modules/ROOT/pages/policy-template-source.adoc

```diff
@@ -0,0 +1,36 @@
+= Policy's Template Source URL
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, CloudFront
+
+[NOTE]
+--
+This article presents configurations parameters thar are only relevant for Mule versions in the ranges [4.1.7, 4.2.0) and [4.2.2,4.3.0).
+--
+
+When an application with an api is deployed to a runtime, the runtime's Api Gateway communicates with Api Manager to ask for the applied policies.
```

Comment:
Not clear what is an application with an API. Please add some more context (eg. application with a tracked flow or similar).

Reply:
thanks for the suggestion, changed.